website_and_docs/content/documentation/webdriver/getting_started/first_script.en.md

```diff
@@ -256,16 +256,38 @@ There are multiple options in each language, but here is what we are using in ou
 
 {{< tabpane code=false langEqualsHeader=true >}}
 {{< tab header="Java" >}}
-// Add instructions
+```shell
+# install Dependent library
+mvn install
+
+# run all case
+mvn test  
+
+#  run FirstScriptTest only   
+# Modify "**/*Test.java" to "**/FirstScriptTest" add Comment out "<include>**/*Example.java</include>" in pom.xml
```

Comment:
Why?

Reply:
Maybe it is better to tell them how we are running the examples?

Reply:
> 

I misunderstood when I thought I only needed to run the FirstScriptTest file

Reply:
> 也许最好告诉他们我们是如何运行示例的？
I'm going to modify it